rfc-002-using-rfcs.md

```diff
@@ -0,0 +1,27 @@
+---
+status: proposed
+---
+
```

Comment:
```suggestion
```
This was a mistake on my part. Adding the status means it ends up in the repository still saying `proposed`, or it needs to say `accepted` in the PR, which doesn't feel like it makes any sense either 🤷 .

I suggest we delete that bit from the template and just say anything in  `main` is accepted, while anything in a branch is proposed.

We can deal with edge cases like revoking decisions if we ever end up with that complexity